src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -117,7 +117,12 @@ def validate_client_parameters(cmd, namespace):
         if not n.account_name:
             n.account_name = get_config_value(cmd, 'storage', 'account', None)
         if auth_mode == 'login':
-            n.token_credential = _create_token_credential(cmd.cli_ctx)
+            if cmd.supported_api_version(min_api='2019-02-02'):
```

Comment:
>if cmd.supported_api_version(min_api='2019-02-02'): [](start = 12, length = 51)

add comment that this check is for track1/track2

Reply:
done